src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -87,12 +59,12 @@ VPC peering is only available for GCP clusters, and AWS PrivateLink is only avai
 1. Navigate to your cluster's **Networking > PrivateLink** tab.
```

Comment:
Shouldn't AWS PrivateLink stuff just link to the new page you've created on that topic?